packages/assets-controllers/src/MultichainTokensRatesController/types.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Represents the rate details for a given token conversion.
+ */
+export type RateData = {
+  conversionTime: number;
+  rate: string;
+};
+
+/**
+ * Represents the conversion rates for a specific token.
+ * The key is a currency identifier (for example, 'swift:0/iso4217:USD'),
+ * and the value is a RateData object.
+ */
+export type TokenConversionRates = {
+  [currency: string]: RateData;
+};
+
+/**
+ * Represents all token conversion rates for a given account.
+ * The key is a token identifier (for example,
+ * 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp/slip44:501'),
+ * and the value is a mapping of currency identifiers to RateData.
+ */
+export type AccountConversionRates = {
+  [tokenId: string]: TokenConversionRates;
```

Comment:
Please use the CAIP types we have rathet than `string` to define these: https://github.com/MetaMask/utils/blob/90837bad1d85277aa5b8f16fe40beb81e0a1e09d/src/caip-types.ts#L119